src/Stratis.Bitcoin.Features.PoA/Voting/FederationVotingController.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Net;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using NBitcoin;
+using Stratis.Bitcoin.Utilities;
+using Stratis.Bitcoin.Utilities.JsonErrors;
+using Stratis.Bitcoin.Utilities.ModelStateErrors;
+
+namespace Stratis.Bitcoin.Features.PoA.Voting
+{
+    [Route("api/[controller]")]
+    public class FederationVotingController : Controller
+    {
+        protected readonly IFederationManager fedManager;
+
+        protected readonly VotingManager votingManager;
+
+        protected readonly Network network;
+
+        protected readonly ILogger logger;
+
+        public FederationVotingController(IFederationManager fedManager, ILoggerFactory loggerFactory, VotingManager votingManager, Network network)
```

Comment:
Shouldn't `GetFederationMembers` also go in this controller?

Reply:
`GetFederationMembers` is in base controller. This method is exactly the same for federation and normal PoA so I saw no reason to put it here.